source/js/raffler.js

```diff
@@ -36,7 +36,13 @@ var Raffler = {
     onKeyDown: function(e) {
         // We only handle the space (32) and page down (34) keys. Page down is enabled because
         // presentation remotes emit page down on the "next" button
-        if (e.keyCode != 32 && e.keyCode != 34) {
+        if (e.keyCode != 32 && e.keyCode != 34 && e.keyCode != 27) {
```

Comment:
Since the list of keys is increasing maybe something like would be more readable:

```js
var acceptedKeys = [27, 32, 34];

if (acceptedKeys.indexOf(e.keyCode) === -1) {
    return;
}
```

We also need to update the comment (to mention something about the `ESC`).